src/cap-description.adoc

```diff
@@ -179,16 +179,27 @@ references if that SDP bit is set because it "owns" that object.
 ==== Sealed (S) Bit
 
 ifdef::cheri_v9_annotations[]
-WARNING: *CHERI v9:* The sealing bit is new (1-bit otype) and the old CHERI v9 otype no longer exists.
+WARNING: *CHERI v9:* The sealing bit is new (1-bit otype) and the old CHERI v9
+otype no longer exists. Please note that this bit indicates the result of what
+was done in two instructions in CHERI v9: CSeal for sealed capabilities and
+CSealEntry for sealed entry capabilties.
 endif::[]
 
-Indicates that a capability is sealed if the bit is 1 or unsealed if it is 0.
+This bit indicates that a capability is sealed if the bit is 1 or unsealed if it is 0.
+
+The sealing bit conflates two concepts in one bit: Sealing data capabilities and
+creating sealed entry (sentry) capabilities.
+
 Sealed capabilities cannot be dereferenced to access memory and are immutable
 such that modifying any of its fields clears the tag of the output capability.
+Sealed capabilities might be useful to software as tokens that can be passed
+around. The only way of removing the seal bit of a capability is by rebuilding
+it via a superset capability with <<CBUILDCAP>>. This architecture does not offer
+an unseal instruction.
 
-In {cheri_base_ext_name}, the sealing bit is used to implement immutable
+For code capabilities, the sealing bit is used to implement immutable
```

Comment:
I think "code capabilities" are sentries. Right? It would be good to consistently use the same language across the descriptions